fontbe/src/features.rs

```diff
@@ -262,24 +262,27 @@ impl<'a> FeatureWriter<'a> {
 
         // Build the actual mark base and mark mark constructs using fea-rs builders
 
-        let mut mark_base_lookups = Vec::new();
-        let mut mark_mark_lookups = Vec::new();
+        let mut mark_lookups = Vec::new();
+        let mut mkmk_lookups = Vec::new();
```

Comment:
I find the prior names clearer. mkmk is particularly obtuse.

Reply:
unfortunately the previous names are inaccurate; these contain lists of lookups for a particular feature (one of `mark` or `mkmk`, but not necessarily only lookups of one type (the `mark` feature contains both mark2base and mark2lig rules)